lib/screens/v2/widget_instance/departures.ex

```diff
@@ -49,15 +49,20 @@ defmodule Screens.V2.WidgetInstance.Departures do
   end
 
   def serialize_section(%{type: :normal_section, rows: departures}, screen) do
-    rows = group_departures(departures)
+    rows =
+      group_departures(departures) ++
+        Enum.filter(departures, &match?(%{text: %Screens.Config.Dup.Override.FreeTextLine{}}, &1))
+
     %{type: :normal_section, rows: Enum.map(rows, &serialize_row(&1, screen))}
   end
 
   def group_departures(departures) do
-    Enum.chunk_by(departures, fn d -> {Departure.route_id(d), Departure.headsign(d)} end)
+    departures
+    |> Enum.reject(&match?(%{text: %Screens.Config.Dup.Override.FreeTextLine{}}, &1))
+    |> Enum.chunk_by(fn d -> {Departure.route_id(d), Departure.headsign(d)} end)
```

Comment:
Instead of treating notice rows differently, you can put each one into its own row by returning a [unique value](https://hexdocs.pm/elixir/Kernel.html#make_ref/0) from the function passed to `Enum.chunk_by`:
```suggestion
    |> Enum.chunk_by(fn
      %{text: FreeTextLine{}} -> make_ref()
      d -> {Departure.route_id(d), Departure.headsign(d)}
    end)
```